src/app/config/config.default.json

```diff
@@ -6,11 +6,9 @@
   "doc-url": "https://glowingbear.app",
   "autosave-subject-sets": false,
   "show-observation-counts": true,
-  "instant-counts-update-1": false,
-  "instant-counts-update-2": false,
-  "instant-counts-update-3": false,
+  "instant-counts-update": false,
   "include-data-table": true,
-  "include-query-subscription": false,
+  "include-cohort-subscription": false,
```

Comment:
there are some inconsistencies in the naming now related to `query` -> `cohort` change, e.g.  here you renamed `include-query-subscription` to `include-cohort-subscription`, but there is still `DEFAULT_INCLUDE_QUERY_SUBSCRIPTION` in src/app/config/app.config.ts

Reply:
also this part of `src/app/config/app.config.ts` should be changed:
```
case 'include-query-subscription': {
          value = AppConfig.DEFAULT_INCLUDE_QUERY_SUBSCRIPTION; break;
        }
````